wgpu-hal/src/lib.rs

```diff
@@ -553,6 +553,15 @@ pub trait Queue: WasmNotSendSync {
 
     /// Submits the command buffers for execution on GPU.
     ///
+    /// Command buffers execute in the order in which they appear in
+    /// `command_buffers`. All results produced by one buffer are visible to the
+    /// next.
+    ///
+    /// If the caller has ensured that one call to `Queue::submit` on a
```

Comment:
/// If `Queue::submit` is called on one `Queue`, and then again on a different queue,
    /// then all the command buffers from the first call will finish executing before any of the 
    /// command buffers from the second call begin.

Reply:
Actually, there's no ordering between queues. I've rephrased to make this clearer.